src/main/java/edu/greenblitz/tobyDetermined/commands/LED/ObjectInClawLED.java

```diff
@@ -0,0 +1,27 @@
+package edu.greenblitz.tobyDetermined.commands.LED;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.util.Color;
+
+
+public class ObjectInClawLED extends LEDCommand{
+
+    public ObjectInClawLED(){
+
```

Comment:
IIRC Java creates an empty constructor automatically and you don't need to declare it yourself.